app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split(" ")
```

Comment:
one space by default 

```suggestion
    words = command.split()
```